tar_scm.py

```diff
@@ -70,8 +70,10 @@ def safe_run(cmd, cwd, interactive=False):
 
 def fetch_upstream_git(url, clone_dir, revision, cwd, kwargs):
     """Fetch sources via git."""
-    safe_run(['git', 'clone', url, clone_dir], cwd=cwd,
-             interactive=sys.stdout.isatty())
+    command = ['git', 'clone', url, clone_dir]
+    if 'sslverify' in kwargs and not kwargs['sslverify']:
```

Comment:
This test is duplicated 4 times; please can you extract into a separate function to adhere to the DRY rule?